docs/UserGuide.md

```diff
@@ -206,7 +206,7 @@ The `delete` command deletes a specified food item from your food inventory.
 
 ## Editing a food item
 
-The edit command edits the details of an existing food item in your food inventory.
+The edit command edits the details of an existing food item in your food inventory. If an entry was incorrect, you can easily edit the entry, without deleting and re-adding the food item.
```

Comment:
Perhaps it should be "If an entry is incorrect, ..."?

Reply:
Changed. Thanks!